website/.storybook/preview.js

```diff
@@ -0,0 +1,20 @@
+import "!style-loader!css-loader!postcss-loader!tailwindcss/tailwind.css";
```

Comment:
this should be replaced in the future with a theme provider there includes chakra ui & tailwind stuff.